src/missions/api.js

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios';
+// mission_id
+// mission_name
+// description
+
+const url = 'https://api.spacexdata.com/v3/missions';
+
+const missions = async () => {
+  const data = await axios.get(url);
+  const formattedBooks = [];
+  Object.keys(data).forEach((key) => {
+    if (key) {
+      formattedBooks.push({
+        ...data[key][0], mission_id: key, mission_name: key, description: key,
+      });
+    }
+  });
+
+  return formattedBooks;
+};
+
+export default missions;
```

Comment:
- [ ] The app will get information from two APIs, one is the `rockets` API, and the other one is the `missions` API. This block of code is just a function creator to update `state.mission`, this means that it should be coded in the `mission.js` file, inside `redux/mission` folder.
- [ ] For this function creator to work, you need the `thunk` dependencies installed. You also need to follow the "function within a function" syntax for `thunkFunctions`. This means you will have one function inside another, and you should pass `dispatch` to the inner one. Like this: `const mission = () => (dispatch) => {/*yourCodeHere*/}`.

> I can see you haven't built the reducer for the mission's interactions. That is something we can do together tomorrow. That way, we could coordinate the best approach to complete the task.